core/domain/user_domain.py

```diff
@@ -801,21 +801,22 @@ class LearnerGoals(python_utils.OBJECT):
     """Domain object for the learner goals model."""
 
     def __init__(
-            self, user_id, topic_ids_to_learn):
+            self, user_id, topic_ids_to_learn,
+            topic_ids_to_master=None):
```

Comment:
Don't have this have a default value. The caller should pass in the correct starting value, e.g. an empty list.

Reply:
Done.